tests/fv3_conf/wam_run.IN

```diff
@@ -1,10 +1,101 @@
-
 inputdir=FV3_input_data_L149
-mkdir INPUT RESTART
-cp -r @[INPUTDATA_ROOT]/${inputdir}/INPUT/* ./INPUT
-cp    @[INPUTDATA_ROOT]/${inputdir}/*grb .
+FV3_FIX=@[INPUTDATA_ROOT]/FV3_fix/
+#WAM_FIX=@[INPUTDATA_ROOT]/${inputdir}/FV3_input_data_wam
```

Comment:
Remove unused, commented line

Reply:
@WChen-NOAA This variable is used further down, did you mean to have it commented out?

Reply:
Sorry, for the late reply.
Those are for the future usage when the whole package goes to ufs-um.